hack/machineset.sh

```diff
@@ -185,7 +185,7 @@ get_vsphere_ms() {
 
   # set golden image template name
   # TODO: read from parameter
-  template="windows-golden-images/windows-server-2022-template"
+  template="windows-golden-images/windows-server-2022-template-containersDisabled"
```

Comment:
I don't think we need this chance, I suggest using `windows-golden-images/windows-server-2022-template` as the latest template name, it's easier to update the template in vCenter.